queries.sql

```diff
@@ -59,4 +59,24 @@ SELECT animals.name FROM animals JOIN owners ON animals.owner_id=owners.id WHERE
 
 SELECT species.name, count(animals) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
 
-SELECT owners.full_name , COUNT(animals.name) AS total_animals FROM owners JOIN animals ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY total_animals DESC LIMIT 1; 
\ No newline at end of file
+SELECT owners.full_name , COUNT(animals.name) AS total_animals FROM owners JOIN animals ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY total_animals DESC LIMIT 1; 
+
+SELECT owners.full_name , COUNT(animals.name) AS total_animals FROM owners JOIN animals ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY total_animals DESC LIMIT 1;
+
+SELECT a.name FROM animals AS a INNER JOIN visits AS j ON j.animals_id = a.id INNER JOIN vets AS v ON j.vets_id = v.id WHERE j.vets_id =1 AND j.date_of_visit = (SELECT MAX(date_of_visit) FROM visits WHERE vets_id = 1);
+
+SELECT COUNT(j.animals_id) FROM visits AS j LEFT JOIN vets AS v ON v.id = j.vets_id WHERE v.id = 3;
+
```

Comment:
* As mentioned by the previous reviewer kindly add a screenshot of your query result as it is a requirement [Check project requirement here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=Take%20a%20screenshot,Pull%20Request%20description.)

Reply:
- [ ] As suggested previously, kindly add screenshots for all queries. You have added only for 4. You have to add for all 9 as per requirements.
<img width="791" alt="Screenshot 2022-07-10 at 1 19 25 AM" src="https://user-images.githubusercontent.com/70269507/178120621-b0160a0a-38bf-47a5-a275-ad734d9bd80f.png">